src/web/routes/application/email-address/email-address.js

```diff
@@ -0,0 +1,31 @@
+// TODO DW HTBHF-1612 add validation
+
+const pageContent = ({ translate }) => ({
+  title: translate('emailAddress.title'),
+  heading: translate('emailAddress.heading'),
+  formDescription: translate('emailAddress.formDescription'),
+  emailAddressLabel: translate('emailAddress.phoneNumberLabel'),
+  buttonText: translate('buttons:continue'),
+  detail: {
+    summaryText: translate('whyDoWeNeedThis'),
+    text: translate('emailAddress.detail.text')
+  }
+})
+
+const contentSummary = (req) => ({
+  key: req.t('emailAddress.summaryKey'),
+  value: `${req.session.claim.emailAddress}`.trim()
```

Comment:
This could just be `req.session.claim.emailAddress.trim()` - without the template string

Reply:
Done.